3.5.0: Build fails on arm64: fatal error: 'asm/hwcap.h' file not found

```
[ 10%] Building CXX object 3rdparty/simdlib/CMakeFiles/visp_simdlib.dir/Simd/SimdBaseCpu.cpp.o
cd /wrkdirs/usr/ports/misc/visp/work/.build/3rdparty/simdlib && /usr/bin/c++ -DSIMD_NEON_FP16_DISABLE -I/wrkdirs/usr/ports/misc/visp/work/visp-3.5.0/3rdparty/simdlib -I/wrkdirs/usr/ports/misc/visp/work/.build/include -O2 -pipe -fstack-protector-strong -fno-strict-aliasing -O2 -pipe -fstack-protector-strong -fno-strict-aliasing -std=gnu++17 -O2 -pipe -fstack-protector-strong -fno-strict-aliasing -fPIC -Wno-asm-operand-widths -Wno-switch -Wno-unused-command-line-argument -MD -MT 3rdparty/simdlib/CMakeFiles/visp_simdlib.dir/Simd/SimdBaseCpu.cpp.o -MF CMakeFiles/visp_simdlib.dir/Simd/SimdBaseCpu.cpp.o.d -o CMakeFiles/visp_simdlib.dir/Simd/SimdBaseCpu.cpp.o -c /wrkdirs/usr/ports/misc/visp/work/visp-3.5.0/3rdparty/simdlib/Simd/SimdBaseCpu.cpp
/wrkdirs/usr/ports/misc/visp/work/visp-3.5.0/3rdparty/simdlib/Simd/SimdBaseCpu.cpp:55:10: fatal error: 'asm/hwcap.h' file not found
#include <asm/hwcap.h>
         ^~~~~~~~~~~~~
1 error generated.
*** [3rdparty/simdlib/CMakeFiles/visp_simdlib.dir/Simd/SimdBaseCpu.cpp.o] Error code 1
```

Failure occurs only on arm64 and not on amd64.

OS: FreeBSD 13

--- a/3rdparty/simdlib/Simd/SimdBaseCpu.cpp
+++ b/3rdparty/simdlib/Simd/SimdBaseCpu.cpp
@@ -52,7 +52,9 @@
 #if !defined(__APPLE__) // Line added for Modification for iOS
 #include <sys/auxv.h>
 #if defined(SIMD_ARM_ENABLE) || defined(SIMD_ARM64_ENABLE)
+#if !defined(__FreeBSD__)
 #include <asm/hwcap.h>
+#endif
 #endif // Line added for Modification for iOS
 #endif
 #endif

--- a/3rdparty/simdlib/Simd/SimdEnable.h
+++ b/3rdparty/simdlib/Simd/SimdEnable.h
@@ -51,7 +51,9 @@
 #include <fcntl.h>
 #include <sys/auxv.h>
 #if defined(SIMD_ARM_ENABLE) || defined(SIMD_ARM64_ENABLE)
+#if !defined(__FreeBSD__)
 #include <asm/hwcap.h>
+#endif
 #endif
 #endif
 #endif

--- a/3rdparty/simdlib/Simd/SimdNeonCpu.cpp
+++ b/3rdparty/simdlib/Simd/SimdNeonCpu.cpp
@@ -28,7 +28,9 @@
 #include <fcntl.h>
 #if !defined(__APPLE__) // Line added for Modification for iOS
 #include <sys/auxv.h>
+#if !defined(__FreeBSD__)
 #include <asm/hwcap.h>
+#endif
 #endif // Line added for Modification for iOS
 #endif
 

